src/test/java/net/rithms/test/sync/ReforgedRuneTest.java

```diff
@@ -0,0 +1,38 @@
+package net.rithms.test.sync;
+
+import net.rithms.riot.api.RiotApi;
+import net.rithms.riot.api.RiotApiException;
+import net.rithms.riot.api.endpoints.static_data.dto.ReforgedRune;
+import net.rithms.riot.constant.Platform;
+import net.rithms.test.RiotApiTest;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ReforgedRuneTest {
+
+    private static RiotApi api = null;
+
+    @BeforeClass
+    public static void prepareApi() {
+        api = RiotApiTest.getRiotApi();
+    }
+
+    @Test
+    public void testGetReforgedRune() throws RiotApiException {
+        ReforgedRune reforgedRune = api.getDataReforgedRune(Platform.EUW, 8014);
```

Comment:
So we have an existing problem right now where the existing tests actually make network calls to the real API and make assertions using real data that could potentially break at anytime, resulting in failed tests. 
Don't get me wrong, I appreciate the fact that you wrote tests for this change and will still accept them as part of the PR. Just wanted to rant/document the fact that I should probably scrap the existing tests and write properly mocked tests for all the endpoints.